servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcUtils.java

```diff
@@ -245,7 +245,7 @@ static GrpcMessageEncoding readGrpcMessageEncoding(final HttpMetaData httpMetaDa
                                                        final Set<GrpcMessageEncoding> allowedEncodings) {
         final CharSequence encoding = httpMetaData.headers().get(GRPC_MESSAGE_ENCODING_KEY);
         if (encoding == null) {
-            return none();
+            return identity();
```

Comment:
followup: I noticed in gRPC we actually encode `identity` on the wire. Since not including the header is equivalent to `identity` we should just not emit identity (for content-encoding or accept-encoding headers). encoding/decoding headers isn't free so we should avoid this when ever possible. as an implementation note we should able to simplify as treating `identity()` the same as `null` (so we don't have to check for both conditions).

Reply:
Will follow up on that on the gRPC PR https://github.com/apple/servicetalk/pull/1198